src/vue/fields/SelectField.vue

```diff
@@ -108,11 +109,19 @@ export default {
         document.addEventListener('click', this.onDocumentClick)
       }
     },
+    values (value) {
```

Comment:
Pls don't use watchers when you can use computed property

Reply:
Moved to computed